### PR TITLE
UserAvatar component

### PR DESCRIPTION
**Pull request by sahilmob**
UserAvatar component

### CODE REVIEW

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  xs: "20px",
+};
+
+const onlineIndicatorStyleMap = {
+  default: {
+    width: "15px",
+    height: "15px",
+    right: "6px",
+    bottom: "-9px",
+    borderWidth: "4px",
```
Comment (ajmnz) [inline kibbeh/src/ui/UserAvatar/SingleUser.tsx:24]: Default size: The red dot (ignoring `border-width`) is `right: 6px` and `bottom: 0`. So taking in account the `border-width`, it should be `right: 2px` and `bottom: -4px`.

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  },
+  sm: {
+    width: "8px",
+    height: "8px",
+    right: "2px",
+    bottom: "-4px",
+    borderWidth: "2px",
```
Comment (ajmnz) [inline kibbeh/src/ui/UserAvatar/SingleUser.tsx:31]: Mid size: The red dot (ignoring `border-width`) is `right: 4px` and `bottom: 0`. So taking in account the `border-width`, it should be `right: 2px` and `bottom: -2px`.

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  },
+  xs: {
+    width: "4px",
+    height: "4px",
+    right: "-1px",
+    bottom: "-1px",
+    borderWidth: "1px",
```
Comment (ajmnz) [inline kibbeh/src/ui/UserAvatar/SingleUser.tsx:38]: Small size: Let's make this one `right: 0px`, `bottom: -1px`.

**Review by ajmnz — CHANGES_REQUESTED** [review]

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx (hunk 1 above)
Comment (sahilmob) [reply kibbeh/src/ui/UserAvatar/SingleUser.tsx:24]: Done

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx (hunk 2 above)
Comment (sahilmob) [reply kibbeh/src/ui/UserAvatar/SingleUser.tsx:31]: Done

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx (hunk 3 above)
Comment (sahilmob) [reply kibbeh/src/ui/UserAvatar/SingleUser.tsx:38]: Done

**Pull request merged**